functests/sctp/sctp.go

```diff
@@ -38,58 +41,136 @@ var _ = Describe("sctp", func() {
 		Expect(err).ToNot(HaveOccurred())
 		err = namespaces.Clean(testNamespace, client.Client)
 		Expect(err).ToNot(HaveOccurred())
-
-		checkForSctpReady(client.Client)
-		By("Creating the sctp service")
-		createSctpService(client.Client)
 	})
-
-	var _ = Context("Client Server Connection", func() {
+	var _ = Describe("sctp setup", func() {
 		var clientNode string
 		var serverNode string
-		It("Should connect a client pod to a server pod", func() {
-			By("Choosing the nodes for the server and the client")
-			nodes, err := client.Client.Nodes().List(metav1.ListOptions{
-				LabelSelector: sctpNodeSelector,
-			})
-			Expect(err).ToNot(HaveOccurred())
-			clientNode = nodes.Items[0].ObjectMeta.Labels[hostnameLabel]
-			serverNode = nodes.Items[0].ObjectMeta.Labels[hostnameLabel]
-			if len(nodes.Items) > 1 {
-				serverNode = nodes.Items[1].ObjectMeta.Labels[hostnameLabel]
-			}
-
-			By("Starting the server")
-			serverArgs := []string{"-ip", "0.0.0.0", "-port", "30101", "-server"}
-			pod := scptTestPod("scptserver", serverNode, "sctpserver", serverArgs)
-			pod.Spec.Containers[0].Ports = []k8sv1.ContainerPort{
-				k8sv1.ContainerPort{
-					Name:          "sctpport",
-					Protocol:      k8sv1.ProtocolSCTP,
-					ContainerPort: 30101,
-				},
-			}
-			serverPod, err := client.Client.Pods(testNamespace).Create(pod)
-			Expect(err).ToNot(HaveOccurred())
 
-			By("Fetching the server's ip address")
-			var runningPod *k8sv1.Pod
-			Eventually(func() k8sv1.PodPhase {
-				runningPod, err = client.Client.Pods(testNamespace).Get(serverPod.Name, metav1.GetOptions{})
+		var _ = Describe("Test Connectivity Negative", func() {
+			beforeAll(func() {
+				By("Validate that SCTP present of cluster.")
+				checkForSctpReady(client.Client)
+			})
+			BeforeEach(func() {
+				By("Choosing the nodes for the server and the client")
+				nodes, err := client.Client.Nodes().List(metav1.ListOptions{
+					LabelSelector: "node-role.kubernetes.io/worker,!node-role.kubernetes.io/worker-sctp",
+				})
 				Expect(err).ToNot(HaveOccurred())
-				return runningPod.Status.Phase
-			}, 3*time.Minute, 1*time.Second).Should(Equal(k8sv1.PodRunning))
+				clientNode = nodes.Items[0].ObjectMeta.Labels[hostnameLabel]
+				serverNode = nodes.Items[0].ObjectMeta.Labels[hostnameLabel]
+				if len(nodes.Items) > 1 {
+					serverNode = nodes.Items[1].ObjectMeta.Labels[hostnameLabel]
+				}
+				createSctpService(client.Client)
+			})
+			Context("Client Server Connection", func() {
+				// OCP-26995
+				It("Negative. SCTP disabled. Should NOT connect a client pod to a server pod", func() {
```

Comment:
Can you put "Negative - Sctp disabled" in the root describe `Describe("Test Connectivity Negative", func() ` and leave only `Should not ...`. It will end up in a more pleasant output